drivers/ethernet/eth_esp32.c

```diff
@@ -288,8 +290,9 @@ static void eth_esp32_iface_init(struct net_if *iface)
 
 	ethernet_init(iface);
 
-	if (device_is_ready(phy_dev)) {
-		phy_link_callback_set(phy_dev, phy_link_state_changed, (void *)dev);
+	if (device_is_ready(eth_esp32_phy_dev)) {
+		phy_link_callback_set(eth_esp32_phy_dev, phy_link_state_changed,
```

Comment:
It fits in a single line, doesn't it?

Reply:
The comma is right on column 80, rest of the driver is mostly on 80 columns as well so I wrapped but happy to bring it back up if you want.

Reply:
Up to you, but Zephyr supports 100 characters in a single line. I'm ok with both options.

Reply:
If I bring it back it'd be the longest line on the file by a shocking 8 columns from the previous longer one. I'm still an 80-column fanboy at heart, not sure I'm emotionally ready to do this if you don't take a strong position about it. :-)

Reply:
No problem at all!